### ISSUE TITLE
install_geographiclib_datasets.sh fails on empty directories

### ISSUE THREAD

**Issue by fnoop** [mavlink/mavros#867]
### Issue details

If download of the geographiclib datasets fails for any reason (like it hangs downloading or can't reach), it leaves empty subdirectories of /usr/share/GeographicLib.  The bash script only checks for existence of these directories, not of the datasets within so any subsequent runs of the script do nothing even if the datasets are not present.

Instead, it should check for the actual files.  Change:
```
	if [[ -d "/usr/share/GeographicLib/$dir" or -d "/usr/local/share/GeographicLib/$dir" ]]; then
		return
	fi
```
 to (eg.)
```
	files=$(shopt -s nullglob dotglob; echo /usr/share/GeographicLib/$dir/$model* /usr/local/share/GeographicLib/$dir/$model*)
	if (( ${#files} )); then
		return
	fi
```

### MAVROS version and platform

Mavros: master
ROS: Lunar
Ubuntu: Debian 9 / Ubuntu 16.04

**Comment by TSC21** [comment]
Thanks for pointing that @fnoop. Would you mind making a PR with this change? Thanks

**Comment by fnoop** [comment]
Sure, PR #868 raised

**Comment by TSC21** [comment]
Closed by #868
